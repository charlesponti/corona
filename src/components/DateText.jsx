import React from 'react'
import PropTypes from 'prop-types'
import Typography from '@material-ui/core/Typography'

const DateText = ({ date }) => (
  <Typography color="textSecondary">{new Date(date).toDateString()}</Typography>
)

DateText.propTypes = {
  date: PropTypes.string
}

export default DateText
