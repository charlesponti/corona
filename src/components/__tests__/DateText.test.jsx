import React from 'react'
import DateText from '../DateText'
import { render } from '@testing-library/react'

test('should render Emoji', () => {
  const { queryByText, container } = render(<DateText date="2020-04-29T00:00:00.000z" />)
  expect(queryByText(/Apr/ig)).toBeTruthy()
  expect(container).toMatchSnapshot()
})
