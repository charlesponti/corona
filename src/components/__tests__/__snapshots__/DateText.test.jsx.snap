// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Emoji 1`] = `
<div>
  <span>
    tue apr 28 2020
  </span>
</div>
`;
